src/shipit_static_analysis/shipit_static_analysis/cli.py

```diff
@@ -106,6 +115,11 @@ def main(phabricator,
                     error=e,
                 )
 
+    # Publish all stats at once
+    end_time = time.time()
+    stats.api.timing('analysis.runtime', end_time - start_time)
+    stats.api.flush(end_time + 5)  # force send last metrics
```

Comment:
Is flushing required? Isn't it done automatically at the end of the execution?
Can you add an explanatory comment?

Reply:
"You know nothing, Jon Snow."

There is a [damn ugly](https://github.com/DataDog/datadogpy/issues/217) bug in the datadog library, and even an outdated [Pull Request](https://github.com/DataDog/datadogpy/pull/221) to fix it (i can't use it for override as it's deprecated)

Reply:
Ahahah, great :)
Can you add this comment to the code? And also file an issue about removing it once the bug is fixed.

Reply:
👍

Reply:
Got a simple fix for that now: they just merged upstream the PR !